include/benchmark/benchmark_api.h

```diff
@@ -154,6 +154,7 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #include <stdint.h>
 
 #include "macros.h"
+#include "benchmark_mpi.h"
```

Comment:
That would make it difficult to create a custom main where MPI_Initialize (MPI_Finalize) are called by outside libraries, whenever those libraries do not separate mpi initialization from other initializations. For instance, it would be difficult to benchmark [Trilinos](https://trilinos.org/).